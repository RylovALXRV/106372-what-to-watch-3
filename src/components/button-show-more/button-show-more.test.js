import React from "react";
import renderer from "react-test-renderer";
import ButtonShowMore from "./button-show-more.jsx";

it(`Render ButtonShowMore`, () => {
  const tree = renderer.create(
      <ButtonShowMore
        onButtonClick={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
