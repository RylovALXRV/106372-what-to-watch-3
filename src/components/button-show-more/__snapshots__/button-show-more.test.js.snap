// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ButtonShowMore 1`] = `
<button
  className="catalog__button"
  onClick={[Function]}
  type="button"
>
  Show more
</button>
`;
